### ISSUE TITLE
Error when installing guzzleBundle

### ISSUE THREAD

**Issue by mkingbe** [symfony/recipes-contrib#145]
Hello,

Cannot install guzzle bundle.

`composer create-project "symfony/skeleton:v4.0.0-BETA2"`

```
composer require eightpoints/guzzle-bundle
Using version ^7.2 for eightpoints/guzzle-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 6 installs, 0 updates, 0 removals
  - Installing symfony/expression-language (v4.0.0-BETA4): Loading from cache
  - Installing guzzlehttp/promises (v1.3.1): Loading from cache
  - Installing psr/http-message (1.0.1): Loading from cache
  - Installing guzzlehttp/psr7 (1.4.2): Loading from cache
  - Installing guzzlehttp/guzzle (6.3.0): Loading from cache
  - Installing eightpoints/guzzle-bundle (v7.2.1): Loading from cache
Writing lock file
Generating autoload files
Symfony operations: 1 recipe (6dd3d05634b6cc068157511b4a9257ca)
  -  WARNING  eightpoints/guzzle-bundle (7.0): From github.com/symfony/recipes-contrib:master
    The recipe for this package comes from the "contrib" repository, which is open to community contributions.
    Do you want to execute this recipe?
    [y] Yes
    [n] No
    [a] Yes for all packages, only for the current installation session
    [p] Yes permanently, never ask again for this project
    (defaults to n): y
  - Configuring eightpoints/guzzle-bundle (7.0): From github.com/symfony/recipes-contrib:master
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 1
!!
!!  In ArrayNode.php line 319:
!!
!!    Unrecognized option "plugin" under "eight_points_guzzle.clients.my_client.options"
!!
!!
!!
```

**Comment by Pierstoval** [comment]
This line should either be removed or untabbed, it seems:

https://github.com/symfony/recipes-contrib/blob/b160c5d377eaf4690d354227ad42b799631bbf29/eightpoints/guzzle-bundle/7.0/config/packages/eight_points_guzzle.yaml#L17

**Comment by mkingbe** [comment]
@gregurco should we remove that line ?

**Comment by gregurco** [comment]
@Pierstoval @mkingbe sorry, my bad. `plugin` option should be on the same level with `options`. I did PR with fix: #150
